src/config.ts

```diff
@@ -12,11 +12,17 @@ export interface VueGtmUseOptions {
     gtm_cookies_win: string;
   };
   /**
-   * Script can be set to `defer` to increase page-load-time at the cost of less accurate results (in case visitor leaves before script is loaded, which is unlikely but possible)
+   * Script can be set to `defer` to decrease page-load-time at the cost of less accurate results (in case visitor leaves before script is loaded, which is unlikely but possible). Defaults to false, so the script is loaded `async` by default
```

Comment:
`"decrease page-load-time" <-> "at the cost of less accurate results"`
This doesn't make sense to me :thinking: 
Why even do set to defer then?

Maybe we should use something more obvious here, like `speed up`

Reply:
Ah yes, speed up sounds good